documentation-site/pages/index.js

```diff
@@ -147,30 +192,12 @@ const Index = (props: {
       </Link>
       , this is one of the fastest solution.
     </Markdown.p>
-    <H2>Who is using Base Web?</H2>
-    <Block>
-      <Block
-        display="grid"
-        gridTemplateColumns="repeat(3, 1fr)"
-        alignItems="center"
-        justifyItems="center"
-        backgroundColor="mono100"
-      >
-        <Block width="125px" $as="img" src="/static/images/uber-logo.png" />
-        <Block width="125px" $as="img" src="/static/images/broadcom-logo.svg" />
-      </Block>
-      <Block
-        $as="a"
-        href="http://t.uber.com/base-web-adopters"
-        $style={{textDecoration: 'none'}}
-        display="block"
-        paddingTop="scale800"
-      >
-        <Button size="compact" kind="secondary">
-          Please let us know if you are using Base Web!
-        </Button>
-      </Block>
-    </Block>
+    <Adopters
+      logoSrcs={[
+        '/static/images/uber-logo.png',
+        '/static/images/broadcom-logo.png',
```

Comment:
what's the reason for changing the broadcom logo to png?

Reply:
@gergelyke 	SVGs don’t scale in IE. See screenshot of IE on the left:
![ie-edge-after-svg](https://user-images.githubusercontent.com/1285326/63139622-da97f400-bf93-11e9-930b-f57fea7793a5.png)